test/ussd_health_worker.test.js

```diff
@@ -177,7 +176,6 @@ describe("familyconnect health worker app", function() {
                     .setup.user.addr('0820000111')
                     .inputs(
                         {session_event: 'new'}  // dial in
-                        , '12345'  // state_auth_code - personnel code
```

Comment:
I'm not sure about the history of the project, but if you look at the list here: https://github.com/praekelt/chms-jsbox/blob/develop/test/fixtures.js#L2-L6 , you can see that all the `111` number is a recognised health worker, so they shouldn't be asked for their personnel code.

I removed this line in all the tests that use the `111` number, because they won't get asked for their code if they're using their phone. I think it must've been there because of some historical thing, but that's not the case for the app anymore.

There are a few tests in the beginning of the test file that deal with whether or not the nurse should have to put in their personnel code, so it's tested there.